src/mount-support.c

```diff
@@ -495,6 +495,16 @@ static void sc_bootstrap_mount_namespace(const struct sc_mount_config *config)
 		die("cannot perform operation: mount --make-rslave %s",
 		    SC_HOSTFS_DIR);
 	}
+	// Detach the hostfs version of sysfs. Some software may want to inspect
+	// the mount table to look for a particular filesystem. If that said
+	// software unlucklily picks the hostfs "version" of. The prominent example
```

Comment:
Typo: 'unlucklily'. This comment is phrased weird. How about:

```
// Detach the redundant hostfs version of sysfs since it shows up in the mount
// table and software inspecting the mount table may become confused (eg,
// docker and LP:# 162601).
```